src/emulator/commandUtils.ts

```diff
@@ -156,6 +156,20 @@ export function parseInspectionPort(options: any): number {
   return parsed;
 }
 
+export function shutdownWhenKilled(): Promise<void> {
+  return new Promise((res, rej) => {
+    process.on("SIGINT", async () => {
```

Comment:
nit: I'd suggest not using `async` for this function, since the promised returned won't be handled by anyone and may hide real errors like below.

Reply:
You're right, let me fix it.